var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -76,11 +76,15 @@ class Hpx(CMakePackage, CudaPackage):
     depends_on('mpi', when='networking=mpi')
 
     # Instrumentation
-    depends_on('apex', when='instrumentation=apex')
+    #depends_on('apex', when='instrumentation=apex')
```

Comment:
This line can go.

Reply:
I left it there in case APEX is not bundled with HPX in the future, but I can remove it if that's not happening anytime soon.

Reply:
Unlikely, unfortunately... that's a bigger task. And the line is easy enough to add back.